### PR TITLE
Bump Dependencies

### PR DESCRIPTION
**Pull request by praveenrewar**
- Bump go to v1.19.4
- Bump golang.org/x/net to v0.4.0

### CODE REVIEW

File: .github/workflows/golangci-lint.yml [hunk 1]
```diff
@@ -12,10 +12,10 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
-      - name: Install Go 1.18
-        uses: actions/setup-go@v3
+      - name: Set up Go
+        uses: actions/setup-go@v2
         with:
-          go-version: "1.18"
```
Comment (100mik) [inline .github/workflows/golangci-lint.yml:18]: Do we want to bump the version of lint being used as well?

Comment (praveenrewar) [reply .github/workflows/golangci-lint.yml:18]: Sure, maybe a separate PR?

Comment (100mik) [reply .github/workflows/golangci-lint.yml:18]: Makes sense since it only affects the action

**Review by 100mik — APPROVED** [review]
LGTM

**Pull request merged**